cmd/kube-controller-manager/app/options/persistentvolumebindercontroller.go

```diff
@@ -57,8 +53,6 @@ func (o *PersistentVolumeBinderControllerOptions) ApplyTo(cfg *persistentvolumec
 
 	cfg.PVClaimBinderSyncPeriod = o.PVClaimBinderSyncPeriod
 	cfg.VolumeConfiguration = o.VolumeConfiguration
-	cfg.VolumeHostCIDRDenylist = o.VolumeHostCIDRDenylist
-	cfg.VolumeHostAllowLocalLoopback = o.VolumeHostAllowLocalLoopback
```

Comment:
shouldn't these fields be removed from cfg type as well?

Reply:
@liggitt removed. please take another look, thank you.